packages/next/build/index.ts

```diff
@@ -863,6 +866,7 @@ export default async function build(dir: string, conf = null): Promise<void> {
       JSON.stringify(prerenderManifest),
       'utf8'
     )
+    await generateClientSsgManifest(prerenderManifest, { distDir, buildId })
```

Comment:
```suggestion
```
We don't need to regenerate it here since we provide the default one in the `build-manifest-plugin` right?

Reply:
Doesn't hurt either way—left the code commented out and added an explanation.